flink-libraries/flink-table/src/main/scala/org/apache/flink/api/java/table/StreamTableEnvironment.scala

```diff
@@ -83,11 +83,54 @@ class StreamTableEnvironment(
       .toArray
 
     val name = createUniqueTableName()
-    registerDataStreamInternal(name, dataStream, exprs)
+    registerDataStreamInternal(name, dataStream, exprs, false)
     ingest(name)
   }
 
   /**
+   * Registers the given [[DataStream]] as table in the
```

Comment:
nitpicking, ScalaDoc indents the `*` under the second `*` of the first line, i.e.,
```
/**
  *
  */
```